app/controllers/default_controller.rb

```diff
@@ -18,6 +18,7 @@ def root
       },
       "res" => {
         "code" => 200,
+        "api_version" => Api::Application::VERSION,
```

Comment:
I'm not sure if this will really be necessary for our needs. We're thinking of this for Orchid, right? A slightly more efficient way to handle it would just be to query the API version with an initializer and set it in a constant.

However, if we are aiming this at general use with stateless API requests where a script/app/whatever won't have knowledge of the API version beforehand and we don't want to require multiple API queries to handle both the version and results, then this is appropriate. I don't know if one or the other is really best practice for API responses in general :thinking:

Reply:
I decided that I kind of like it as a forward thinking step. Ultimately, the code I wrote for Orchid didn't need to manually check it, since it can tell from the facets response how it should treat everything.  But thinking of things besides Orchid, it might be nice to know with each returning request which version of the API it is.  Truly, if we want to be a bit more like some of the big APIs that I used to query at SEMCAT, we would also returning information about like, the datetime when the response was sent, etc, which I would be happy to add here as well.  It seems like it would be useful information.

Reply:
Okay. That sounds fine to me :+1:

Reply:
That's an interesting idea. Usually I think there was a "requested_time" and a "response_time" kind of situation, so you could have calculated it yourself if you wanted to, but it could be fun to add an interval....

Reply:
Might as well provide all three and save people (including future us) the hassle :)

Reply:
How would you feel about a new top level part of the response?

```
info: { timestamps, version, etc }
req: { what they sent }
res: { documents and facets }
```

I like the idea of separating it out if we're going to be including more stuff there.  I don't know what it should be called, though.  Info? Meta?

Reply:
I think I said in our standup yesterday that I think `res: { meta: { ... } }` would be a good approach. Info or meta seem fine. I just wonder if `info` is more likely to be used for something else in the future :man_shrugging:

Reply:
sounds good!

Reply:
Looks like the collections endpoint is already using "info" although we can redo that right now, too, while we changing how the response comes back

Reply:
I added this line to the "info" sub-hash